src/main/kotlin/org/jetbrains/intellij/tasks/SignPluginTask.kt

```diff
@@ -0,0 +1,46 @@
+package org.jetbrains.intellij.tasks
+
+
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.internal.ConventionTask
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.*
+import org.jetbrains.zip.signer.signer.CertificateUtils
+import org.jetbrains.zip.signer.signer.PrivateKeyUtils
+import org.jetbrains.zip.signer.signer.PublicKeyUtils
+import org.jetbrains.zip.signer.signing.DefaultSignatureProvider
+import org.jetbrains.zip.signer.signing.ZipSigner
+
+open class SignPluginTask : ConventionTask() {
```

Comment:
can we have kdoc on all properties? so it's readily available during build file editing in IDE